pipelines/upload_transacao_cct/tasks.py

```
@@ -0,0 +1,293 @@
+                    FROM STDIN WITH CSV HEADER
+                """
+
+                with blob.open("r") as f:
+                    cur.copy_expert(sql, f)
+                log("Cópia completa")
+
+                sql = f"""
+                    DELETE FROM public.{table_name} t
+                    USING public.{tmp_table_name} s
+                    WHERE t.id_transacao = s.id_transacao
+                """
```

Ruff (0.14.0)

Possible SQL injection vector through string-based query construction

(S608)